### PR TITLE
RoR2: bug fixes

### PR DESCRIPTION
**Pull request by kindasneaki**
## What is this fixing or adding?
Fix 2 maps ids being backwards
Adding in a missing item for one of the victory's
re added parenthesis for lunartic and void presets to generate correctly

## How was this tested?
Ran generate.py multiple times.

### CODE REVIEW

File: worlds/ror2/__init__.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class RiskOfRainWorld(World):
     location_name_to_id = item_pickups
 
     data_version = 6
```
Comment (ThePhar) [inline worlds/ror2/__init__.py:38]: You're adding a new item to your item table, so you should increment this.

```suggestion
    data_version = 7
```

File: worlds/ror2/__init__.py [hunk 2]
```diff
@@ -113,22 +113,23 @@ def create_items(self) -> None:
...
         # Generate item pool
         itempool: List = []
         # Add revive items for the player
         itempool += ["Dio's Best Friend"] * self.total_revivals
+        itempool += ["Beads of Fealty"]
 
         for env_name, _ in environments_pool.items():
             itempool += [env_name]
 
         # precollected environments are popped from the pool so counting like this is valid
```
Comment (ThePhar) [inline worlds/ror2/__init__.py:131]: Is this comment still valid?

**Review by ThePhar — CHANGES_REQUESTED** [review]

File: worlds/ror2/__init__.py (hunk 1 above)
Comment (kindasneaki) [reply worlds/ror2/__init__.py:38]: I thought this was deprecated?

File: worlds/ror2/__init__.py (hunk 2 above)
Comment (kindasneaki) [reply worlds/ror2/__init__.py]: No, thanks for pointing it out

File: worlds/ror2/__init__.py (hunk 1 above)
Comment (ThePhar) [reply worlds/ror2/__init__.py]: Older clients haven't been dropped completely yet, so we still support it... for now.

**Review by ThePhar — APPROVED** [review]

**Pull request merged**